seed/fern-cli/seed.yml

```diff
@@ -1 +1,25 @@
 changelogLocation: ../../packages/cli/cli/versions.yml
+publishGa:
+  versionSubstitution: $VERSION
+  command:
+    - cd packages/cli/cli
+    - mv package.json package.json.tmp
+    - version_replace="s/0.0.0/$VERSION/"
+    - cat package.json.tmp| sed "$VERSION" > package.json
+    - rm -rf package.json.tmp
+    - pnpm install
+    - pnpm --filter @fern-api/cli compile
+    # Should likely include this within the script itself
+    - pnpm --filter @fern-api/cli publish:cli:prod --tag latest
+publishRc:
```

Comment:
should we move over `publishDev` as well since that will allow us to to test this more easily

Reply:
chatted offline, needs seed-dev or if too hard, just don't register (but still publish)

Reply:
Updated